spec/controllers/course/enrol_requests_controller_spec.rb

```diff
@@ -0,0 +1,144 @@
+require 'rails_helper'
+
+RSpec.describe Course::EnrolRequestsController, :type => :controller do
+  before do
+    ActsAsTenant.current_tenant = Instance.default
+  end
+
+  let!(:user) { create(:user) }
+  let!(:course) { create(:course, status: :opened) }
+  let!(:student_request) { create(:course_enrol_request, course: course, user: create(:user),
+                                  role: :student) }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 11, 51 should be on its own line.